GitUp/Application/AppDelegate.m

```diff
@@ -411,6 +413,11 @@ - (void)applicationDidFinishLaunching:(NSNotification*)notification {
 #endif
 }
 
+- (void)_getUrl:(NSAppleEventDescriptor*)event withReplyEvent:(NSAppleEventDescriptor*)replyEvent {
+  NSString* url = [[event paramDescriptorForKeyword:keyDirectObject].stringValue stringByReplacingOccurrencesOfString:@"github-mac://openRepo/" withString:@""];
```

Comment:
I mostly just took the info from here: https://gist.github.com/mralexgray/3915592
I'll see if I can simplify the call as I do agree it's sort of convoluted.